src/combat.ts

```diff
@@ -668,41 +693,41 @@ export class Macro extends StrictMacro {
       Macro.if_($location`The Briny Deeps`, Macro.tryCopier($item`pulled green taffy`))
         .externalIf(
           myFamiliar() === $familiar`Reanimated Reanimator`,
-          Macro.trySkill($skill`Wink at`),
+          Macro.trySkill($skill`Wink at`)
         )
         .externalIf(
           myFamiliar() === $familiar`Obtuse Angel`,
-          Macro.trySkill($skill`Fire a badly romantic arrow`),
+          Macro.trySkill($skill`Fire a badly romantic arrow`)
         )
         .externalIf(
           doneHabitat &&
             get("beGregariousCharges") > 0 &&
             (get("beGregariousMonster") !== embezzler ||
               get("beGregariousFightsLeft") === (have($item`miniature crystal ball`) ? 0 : 1)),
-          Macro.trySkill($skill`Be Gregarious`),
+          Macro.trySkill($skill`Be Gregarious`)
         )
         .externalIf(
           have($skill`Just the Facts`) &&
             get("_monsterHabitatsRecalled") < 3 &&
             (get("monsterHabitatsMonster") !== embezzler ||
               get("monsterHabitatsFightsLeft") === (have($item`miniature crystal ball`) ? 0 : 1)),
-          Macro.trySkill($skill`Recall Facts: Monster Habitats`),
+          Macro.trySkill($skill`Recall Facts: Monster Habitats`)
         )
         .externalIf(
           have($skill`Recall Facts: %phylum Circadian Rhythms`) &&
             !get("_circadianRhythmsRecalled"),
-          Macro.trySkill($skill`Recall Facts: %phylum Circadian Rhythms`),
+          Macro.trySkill($skill`Recall Facts: %phylum Circadian Rhythms`)
         )
         .externalIf(
           SourceTerminal.getDigitizeMonster() !== embezzler || shouldRedigitize(),
-          Macro.tryCopier($skill`Digitize`),
+          Macro.tryCopier($skill`Digitize`)
         )
         .tryCopier($item`Spooky Putty sheet`)
         .tryCopier($item`Rain-Doh black box`)
         .tryCopier($item`4-d camera`)
         .tryCopier($item`unfinished ice sculpture`)
         .externalIf(get("_enamorangs") === 0, Macro.tryCopier($item`LOV Enamorang`))
-        .meatKill(),
+        .meatKill()
```

Comment:
It seems like we may have a linting rule that is being inconsistently applied across different installs

Reply:
This is due to [prettier 3.0](https://prettier.io/blog/2023/07/05/3.0.0.html)

> Change the default value for `trailingComma` to `all`